tt-train CI failure

https://github.com/tenstorrent/tt-metal/actions/runs/25570907644/job/75083746196?pr=43745

```
Failed to generate binaries for sdpa_fw_compute_kernel TT_THROW @ /work/tt_metal/jit_build/build.cpp:62: tt::exception
info:
trisc1 build failed. Log: In file included from /work/tt-train/sources/ttml/metal/ops/sdpa_fw/device/kernels/compute/sdpa_fw_compute_kernel.cpp:30,
                 from ../chlkc_math.cpp:3,
                 from /work/tt_metal/hw/ckernels/wormhole_b0/metal/common/chlkc_list.h:16,
                 from /work/tt_metal/hw/firmware/src/tt-1xx/trisck.cc:13:
/work/tt-train/sources/ttml/metal/ops/sdpa_fw/device/kernels/compute/sdpa_compute_utils.hpp: In function 'void calculate_recip_first_column()':
/work/tt-train/sources/ttml/metal/ops/sdpa_fw/device/kernels/compute/sdpa_compute_utils.hpp:38:90: error: cannot convert 'int' to 'sfpi::RoundMode'
   38 |             sfpi::dst_reg[0] = sfpi::reinterpret<sfpi::vFloat>(sfpi::float_to_fp16b(out, 0));
      |                                                                                          ^
      |                                                                                          |
      |                                                                                          int
In file included from /work/runtime/sfpi/include/sfpi.h:551,
                 from /work/tt_metal/tt-llk/tt_llk_wormhole_b0/common/inc/ckernel_sfpu.h:12,
                 from /work/tt_metal/tt-llk/tt_llk_wormhole_b0/common/inc/cmath_common.h:13,
                 from /work/tt_metal/hw/ckernels/wormhole_b0/metal/llk_api/llk_math_common_api.h:10,
                 from /work/tt_metal/hw/inc/api/compute/common_globals.h:16,
                 from /work/tt_metal/hw/inc/api/compute/cb_api.h:7,
                 from /work/tt-train/sources/ttml/metal/ops/sdpa_fw/device/kernels/compute/sdpa_fw_compute_kernel.cpp:5:
/work/runtime/sfpi/include/sfpi_lib.h:282:57: note: initializing argument 2 of 'sfpi::vFloat sfpi::float_to_fp16b(vFloat, RoundMode)'
  282 | sfpi_inline vFloat float_to_fp16b (vFloat in, RoundMode rounding = RoundMode::Stochastic) {
      |                                               ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```

--- a/tt-train/sources/ttml/metal/ops/sdpa_fw/device/kernels/compute/sdpa_compute_utils.hpp
+++ b/tt-train/sources/ttml/metal/ops/sdpa_fw/device/kernels/compute/sdpa_compute_utils.hpp
@@ -35,7 +35,7 @@ void calculate_recip_first_column() {
             sfpi::dst_reg[0] = ckernel::sfpu::_sfpu_reciprocal_<2>(in);
         } else {
             sfpi::vFloat out = ckernel::sfpu::_sfpu_reciprocal_<1>(in);
-            sfpi::dst_reg[0] = sfpi::reinterpret<sfpi::vFloat>(sfpi::float_to_fp16b(out, 0));
+            sfpi::dst_reg[0] = sfpi::float_to_fp16b(out, sfpi::RoundMode::NearestEven);
         }
         sfpi::dst_reg += 2;
     }

